benches/packet_processing.rs

```diff
@@ -24,7 +24,7 @@ fn process_packet_before_send(
 fn send_unreliable_benchmark(c: &mut Criterion) {
     let config = Config::default();
     let mut connection =
-        VirtualConnection::new(SERVER_ADDR.parse().unwrap(), Arc::new(Config::default()));
+        VirtualConnection::new(SERVER_ADDR.parse().unwrap(), &Arc::new(Config::default()));
```

Comment:
Looking at this now, it looks like we either need to rewrite these benches or delete them completely. They don't run in CI and are broken when run with `cargo bench`.

Reply:
Deleting them. We can write new ones after @TimonPost 's PR and this are merged